### PR TITLE
[kvdb-rocksdb] add DatabaseConfig max_total_wal_size

### PR DESCRIPTION
**Pull request by mdben1247**
It would be useful to be able to limit the disk usage of write ahead logs. The default value set up by `rocksdb` can be rather large, in some `openethereum` nodes I am testing with it's about 50GB or more, considerably raising disk space requirements.

### CODE REVIEW

File: kvdb-rocksdb/src/lib.rs [hunk 1]
```diff
@@ -177,6 +177,9 @@ pub struct DatabaseConfig {
 	/// if the secondary instance reads and applies state changes before the primary instance compacts them.
 	/// More info: https://github.com/facebook/rocksdb/wiki/Secondary-instance
 	pub secondary: Option<String>,
+	/// Limit the size of write ahead logs
```
Comment (ordian) [inline kvdb-rocksdb/src/lib.rs:180]:
```suggestion
	/// Limit the size (in bytes) of write ahead logs
```

**Review by ordian — APPROVED** [review]

**Comment by ordian** [comment]
please fix the test as well

**Review by dvdplm — APPROVED** [review]

**Comment by ordian** [comment]
`cargo fmt` should make the CI green

**Comment by mdben1247** [comment]
Sorry, I am a bit new to this. Should I run this `cargo fmt` with some options?

**Comment by ordian** [comment]
> Sorry, I am a bit new to this. Should I run this `cargo fmt` with some options?

No worries! No, should be good as is.

**Pull request merged**